.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: java
-deploy:
-  provider: heroku
-  api_key:
-    secure: fQaUc06agYL1mMjydOzTCO8q6vOMcLLB98OFzAVy24BzZJU+npMNPVTv8g9CvUa/tyoKkhlSojKHxKp5tUXoArP+CNOwSfYq56IdOPNHxIlHiwqFD/7JBkpNHlSTdCoy5f4+JzlVSa3ryxf6ovPBpjQGVQR8rCc4ApO5L1kTNq1dVbdDHX68nyQiuVoXcTAs6F/JqdeyaI/UBY9prWP5IU1xWnfKH8YPGNPx3gnhkjnIgR/tae4zqrYPUTblfds5xuk6eAxTQrwPa/CBQxoaNTxQ1U6+rxIdGR0hfruL6qDgXYsXdX3Ua7fUiU57hl9ggcTgDNrxMGGZ6FRy31n3RkNJu3wSg5IhUdTEjoP0uV4L15kMtNgtPI5zK0cupPKEWBHs9hKgeHx2LDIhV1hR/LouCMUTe51iZa17GC9ZOSlP7zvgy29U94bgD12w55N06+dN3q27i+/nXanBipx1m98X3Gjqvkw011fxN4gvK3NzX3saXK+UYiGzks2Jut8ct1rmKvj3Lt0m79wsbm5GILvToLhnDNbTFOmjar3wzwszuHAIzZbZrZKG5cf5/M59qCyGk1IGYUx4Ht0vc9+rVugSkDdNksDAGz9sX0Sj2UlWBVrI5GQ0grsTBQGLVpLLDde7Ud+95eBa6ImzzhEBWxZLbejZZK8pcaWhwRLH2ZE=
-  app: floating-peak-57859
-  on:
-    repo: serpient/java_http_server
+# deploy:
+#   provider: heroku
+#   api_key:
+#     secure: fQaUc06agYL1mMjydOzTCO8q6vOMcLLB98OFzAVy24BzZJU+npMNPVTv8g9CvUa/tyoKkhlSojKHxKp5tUXoArP+CNOwSfYq56IdOPNHxIlHiwqFD/7JBkpNHlSTdCoy5f4+JzlVSa3ryxf6ovPBpjQGVQR8rCc4ApO5L1kTNq1dVbdDHX68nyQiuVoXcTAs6F/JqdeyaI/UBY9prWP5IU1xWnfKH8YPGNPx3gnhkjnIgR/tae4zqrYPUTblfds5xuk6eAxTQrwPa/CBQxoaNTxQ1U6+rxIdGR0hfruL6qDgXYsXdX3Ua7fUiU57hl9ggcTgDNrxMGGZ6FRy31n3RkNJu3wSg5IhUdTEjoP0uV4L15kMtNgtPI5zK0cupPKEWBHs9hKgeHx2LDIhV1hR/LouCMUTe51iZa17GC9ZOSlP7zvgy29U94bgD12w55N06+dN3q27i+/nXanBipx1m98X3Gjqvkw011fxN4gvK3NzX3saXK+UYiGzks2Jut8ct1rmKvj3Lt0m79wsbm5GILvToLhnDNbTFOmjar3wzwszuHAIzZbZrZKG5cf5/M59qCyGk1IGYUx4Ht0vc9+rVugSkDdNksDAGz9sX0Sj2UlWBVrI5GQ0grsTBQGLVpLLDde7Ud+95eBa6ImzzhEBWxZLbejZZK8pcaWhwRLH2ZE=
```

Comment:
I don't know what this is, but you shouldn't ever commit secure api keys. Once you commit it, it's available for anyone to see forever.

Reply:
This is a encrypted key that has been generated. Is that still bad to commit?

Reply:
For toy projects it's probably iffy-but-ok, but I'd say that it's a good habit to avoid because it's not ever okay to do with a client.  Once something hits your git history, it more or less stays there - meaning if there's ever an insecure thing committed and pushed upstream then it's permanently out there even if you add another commit removing it.

Reply:
The fact that it's an encrypted key is irrelevant - what matters is that it's a key that should be kept private (any key that is described as secure should not be shared)